docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -0,0 +1,154 @@
+## Get started with {{% param "PRODUCT_NAME" %}}
```

Comment:
```suggestion
---
canonical: https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/
description: Learn how to send metrics to Prometheus
menuTitle: Send metrics to Prometheus
title: Send 
weight: 15
---
# Send metrics to Prometheus
```

Reply:
I think the title and the menuTitle may be swapped in this suggestion though I also think the menuTitle being "Send" is maybe a bit too short for discovery in the ToC?

I've not looked at anything else in the PR.

Reply:
Oh, my mistake there... a copy/paste ooops. I'll fix it up in the suggestion. The final tutorial title still needs to be decided upon.